docs/dev/resources/networking.md

```diff
@@ -0,0 +1,57 @@
+# Networking Common Tasks and Information
+
+## Quick Access
+[Allow Connection to Sites on the Internet from a ProB workload](#allowing-connection-to-sites-on-the-internet)
+
+[Allow Connection to internal application](#allow-connection-to-internal-application)
+
+[DNS pitfalls](#dns-pitfalls-to-internal-application-connecting)
+
+[The `Priority` field in the fwprcg](#the-priority-field)
+
+[In which repo should we define our resources?](#in-which-repository-should-we-define-the-resources)
+
+----
+### Allowing Connection to sites on the Internet
+This is a bit more niche, as our unclassified notebooks already have connection to the internet, but this can come up when we have a Protected B certified application like seen in https://github.com/StatCan/aaw-private/issues/125 where it's accessible to the internet, but unaccecssible from our Protected B workloads.
+
+To handle this we want to declare a [firewall_policy_rule_collection_group](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/firewall_policy_rule_collection_group) in our terraform files. 
+
+Regarding certain fields, the rule collection group type we chose was [`application rule`](https://learn.microsoft.com/en-us/azure/firewall/policy-rule-sets#application-rules) as we are filtering based on FQDNs.
+
+The `source address` is the subnet address of our protected-b node pool. 
+
+-------------
+### Allow Connection to internal application in an already peered network

```

Comment:
You can just add a section on peering, since the rest of the steps are all the same, there is just one additional step to define the network peering on each end!

Reply:
Yup added a tiny section on peering